README.md

```diff
@@ -114,6 +134,42 @@ normalized_df = pycytominer.normalize(
 )
 ```
 
+### Handling Non-CellProfiler Morphological Features in Pycytominer
+
+In some cases, raw morphological features may not be extracted from [CellProfiler](https://cellprofiler.org/).
+While Pycytominer fully supports features extracted by [`CellProfiler`](https://cellprofiler.org/), errors may occur when using features from other tools.
+
+To resolve this, you can manually specify the morphological features using the `features` [parameter](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#pycytominer.normalize.normalize).
+This parameter is also available in other key steps, such as [`aggregate`](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#pycytominer.aggregate.aggregate), [`feature_select`](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#pycytominer.feature_select.feature_select), and [`consensus`](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#pycytominer.feature_select.feature_select).
+
+Below is an example of loading data that is not from [`CellProfiler`](https://cellprofiler.org/), demonstrating how to handle non-CellProfiler features.
+
+```python
+# Real world example using a different dataset
```

Comment:
@axiomcura - maybe the reason why this is marked as a draft, but this example _is_ using a CellProfiler output

Reply:
You’re right, the code and link provided does point to CellProfiler. I was considering adding a comment like ‘assuming this dataset was not generated by CellProfiler,’ but in hindsight, I think this might still end up confusing users.

Are there any datasets available that weren’t generated by CellProfiler? Unfortunately, the InCarta data is private, so we can’t use it as an example. 

I can also remove those links and add placeholders like ‘link to non-CellProfiler data,’ which we can update once we have an appropriate link available.